.github/workflows/collector-builder.yml

```diff
@@ -8,10 +8,20 @@ on:
         required: true
         description: |
           The tag used to build the collector image
+      architectures:
+        type: string
+        required: false
+        description: |
+          Space-seperated list of architectures
+        default: |
+          ${{ ((github.event_name != 'pull_request' ||
+            contains(github.event.pull_request.labels.*.name, 'run-multiarch-builds')) &&
+            'amd64 arm64 ppc64le s390x') || 'amd64 arm64' }}
```

Comment:
I see this logic is duplicated for `collector.yml` and `collector-builder.yml`, yet the input is not actually used.

Consider making this logic an output of the `init.yml` workflow, then pass it in to these 2 workflows in `main.yml`. Also change the `required` field to `true` to ensure it is properly being provided.

See how the `collector-tag` variable is handled for an example on how to do this.